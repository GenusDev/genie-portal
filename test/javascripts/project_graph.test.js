import ProjectGraph from '../../frontend/components/entities/dashboard/project_dashboard/project_graph'

function componentSetup(){
  const props = {
    fetchProjects: jest.fn(()=>{
      return Promise.resolve();
    })
  }
  const eWrapper = mount(<ProjectGraph {...props}/>)
  return {
    props,
    eWrapper
  }
}


it('should match snapshot', () => {
     const { eWrapper} = componentSetup()
     // expect(eWrapper.find('section').hasClass('graph-container')).toBe(true)
      expect(eWrapper).toMatchSnapshot();
   })
