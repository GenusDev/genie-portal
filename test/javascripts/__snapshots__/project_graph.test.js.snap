// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Graph Test should match snapshot 1`] = `
<div
  className="graph-container"
>
  <div
    className="series content graph"
    id="project"
  >
    <div
      id="graph"
    />
  </div>
</div>
`;
